src/plugins/intel_cpu/src/plugin.cpp

```diff
@@ -292,6 +292,11 @@ void Engine::GetPerformanceStreams(Config& config, const std::shared_ptr<ngraph:
     const auto latency_name = std::string(CONFIG_VALUE(LATENCY)) + "_" + std::string(ov::num_streams.name());
     const auto tput_name = std::string(CONFIG_VALUE(THROUGHPUT)) + "_" + std::string(ov::num_streams.name());
 
+#if defined(OPENVINO_ARCH_ARM) || defined(OPENVINO_ARCH_ARM64)
```

Comment:
We already force num_streams to be equal 1 in CPU config. Why it should be updated here?

Reply:
This WA for CI checks will be removed after export/import refactor for MT 2.0 which will be track in CVS-112149

Reply:
Please put this sentence as comment in the code

Reply:
updated